MVP guidelines for Slider (#2449)
coveo/plasma: packages/website/docs/Slider.md

Use sliders carefully, as they can be difficult to control on smaller devices or when the range is very large but the space is limited.

## Labeling

Keep marker labels short, preferably **under only one word**.

Provide a short descriptive title without action verbs. For example, write "Thumbnail size" rather than "Select the size of the thumbnail".

## Track and markers

The track is the representation of the range. It should be divided into equal steps using markers.

Name markers using adjectives when exposing the numeric value is unimportant. For instance, use “small” and “big” when choosing the size of a thumbnail.

When using numeric value to label markers, display the value unit to provide context over the type of value. For instance, show "90%" if setting a percentage. 

Use positive and negative symbols as prefixes when the range allows negative values.

## Feedback and Validation

A slider always has a default value. Consider the most frequently set value as default first. If none particularly stands out, consider using a neutral value or the median. Note that the default value should stand on a marker as much as possible.

> If a **numeric** value between two markers can be chosen, display the selected value above the handle or next to the slider so users can easily refer to what they have selected.

Reviewer: ```suggestion
If users can select a numeric value between two markers, display this value above the handle or next to the slider. This allows users to refer to what they selected.
```

---

## Related Components

> If your use case doesn't match the guidelines above, consider using one of the following components instead:

Reviewer: ```suggestion
If setting a precise value is important, consider using a [numeric input](#/form/NumericInput) instead.
```

Reviewer 2: @kpoulincoveo

> -   [Numeric input](#/form/NumericInput) when setting a precise value is important.

Reviewer: If you accepted my previous comment, delete this.

Reviewer 2: @kpoulincoveo

Author: Accepted @aiheon ready to merge
